### PR TITLE
introduce github actions

### PR DESCRIPTION
**Pull request by xetys**

### CODE REVIEW

**Comment by squ94wk** [comment]
please rebase onto master. I just merged the Dockerfile MR.

File: go.mod [hunk 1]
```diff
@@ -4,11 +4,107 @@ go 1.17
...
+	github.com/beorn7/perks v1.0.1 // indirect
+	github.com/blang/semver v3.5.1+incompatible // indirect
+	github.com/cespare/xxhash/v2 v2.1.2 // indirect
+	github.com/coreos/go-semver v0.3.0 // indirect
+	github.com/coreos/go-systemd/v22 v22.3.2 // indirect
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/emicklei/go-restful v2.9.5+incompatible // indirect
+	github.com/evanphx/json-patch v4.12.0+incompatible // indirect
+	github.com/felixge/httpsnoop v1.0.1 // indirect
+	github.com/fsnotify/fsnotify v1.5.1 // indirect
```
Comment (furkhat) [inline go.mod:26]: it looks like it needs `go mod tidy`

File: main.go [hunk 2]
```diff
@@ -2,6 +2,7 @@ package main
 
 import (
 	"fmt"
+	"github.com/cert-manager/cert-manager/pkg/acme/webhook"
```
Comment (furkhat) [inline main.go:5]: Let's move this import down so it is next to other `cert-manager` imports

**Review by furkhat — CHANGES_REQUESTED** [review]
two minor things, overall LGTM

File: go.mod (hunk 1 above)
Comment (xetys) [reply go.mod:26]: it doesn't change if I do that

**Review by furkhat — APPROVED** [review]

**Pull request merged**